Tests/SPMTest/UISPMTest/UISPMTest.xcodeproj/project.pbxproj

```diff
@@ -0,0 +1,363 @@
+// !$*UTF8*$!
```

Comment:
Should we be checking in an Xcode project that’s generated by xcodegen based on a variable that changes dynamically?

Reply:
Good idea, but I'm not sure that there is any easy way to check what version is currently used in generated Xcode project (maybe another 3rdparty tool like `XcodeProj` can do that). I'll do some more tests to check if we can do this in scope of this PR.